src/flask_pyoidc/pyoidc_facade.py

```diff
@@ -104,42 +105,51 @@ def authentication_request(self, state, nonce, extra_auth_params):
     def login_url(self, auth_request):
         """
         Args:
-            auth_request (AuthorizationRequest): authenticatio request
+            auth_request (AuthorizationRequest): authentication request
         Returns:
             str: Authentication request as a URL to redirect the user to the provider.
         """
         return auth_request.request(self._client.authorization_endpoint)
 
     def parse_authentication_response(self, response_params):
         """
-        Args:
-            response_params (Mapping[str, str]): authentication response parameters
-        Returns:
-            Union[AuthorizationResponse, AuthorizationErrorResponse]: The parsed authorization response
+        Parameters
+        ----------
+        response_params: Mapping[str, str]
+            authentication response parameters.
+
+        Returns
+        -------
+        Union[AuthorizationResponse, AuthorizationErrorResponse]
+            The parsed authorization response.
         """
-        auth_resp = self._parse_response(response_params, AuthorizationResponse, AuthorizationErrorResponse)
+        auth_resp = self._client.parse_response(AuthorizationResponse, info=response_params, sformat='dict')
```

Comment:
This is the part that will not work without https://github.com/OpenIDC/pyoidc/pull/805
(Just adding note to remember. 🙂 )

Reply:
Resolved. OpenIDC/pyoidc#805 has been merged.